src/test/java/Homework16.java

```diff
@@ -0,0 +1,59 @@
+import org.checkerframework.checker.units.qual.C;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 {
+
+    @Test
+    public static void createNewPlaylist () throws InterruptedException {
```

Comment:
public  void createNewPlaylist ()  
do not use statics unnecessarily

Reply:
It kept making me use static. Is there a reason why it would?